site/content/docs/11-deploying.md

```diff
@@ -4,28 +4,25 @@ title: Deployment
 
 Sapper apps run anywhere that supports Node 8 or higher.
 
+### Deploying to Vercel ([formerly ZEIT Now](https://vercel.com/blog/zeit-is-now-vercel))
 
-### Deploying to Now
+#### Using Now v2
```

Comment:
I think we decided that this should say "Vercel" instead of "Now v2", right?

Reply:
@diverent2 just checking in to see if you'd be able to update this line. That's the last comment I have on this PR. Thanks!